app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesActivity.java

```diff
@@ -121,34 +111,39 @@ public void checkInAlertBuilder(final int index) {
             + "Ticket: " + thisAttendee.getTicket().getType())
             .setPositiveButton("OK", (dialog, which) -> {
                 Log.d(TAG, "onClick: inside ok");
-                changeCheckStatus(thisAttendee.getId(), index);
+                attendeesPresenter.toggleAttendeeCheckStatus(index, thisAttendee.getId());
```

Comment:
Why passing the index? Pass the attendee to operate on instead

Index is extremely unreliable and will be difficult to track bugs when we implement sorting and filtering in future